debug/org.eclipse.debug.ui/ui/org/eclipse/debug/internal/ui/views/launch/DebugElementHelper.java

```diff
@@ -44,7 +44,7 @@ public class DebugElementHelper {
 	private static DelegatingModelPresentation fgPresenetation;
 
 	// map of images to image descriptors
-	private static Map<Image, ImageDescriptor> fgImages = new HashMap<>();
+	private static Map<Image, ImageDescriptor> fgImages = new ConcurrentSkipListMap<>();
```

Comment:
Why not `ConcurrentHashMap`? Any specific order needed here? It wasn't sorted before.

Reply:
Now it is green. 
Sorry for wasting your time on red PR. Please let me know if you prefer `ConcurrentHashMap`

Reply:
Thanks.